product_docs/docs/pgd/5/upgrades/upgrade_paths.mdx

```diff
@@ -2,28 +2,26 @@
 title: Supported PGD upgrade paths
 ---
 
-
 ## Upgrading within version 5
 
-| 5.0.0 | 5.0.1 | 5.1.0 | Target PGD version |
-|-------|-------|-------|--------------------|
-| ✓     | ✓     | ✓     | 5.2.0              |
-| ✓     | ✓     |       | 5.1.0              |
-| ✓     |       |       | 5.0.1              |
-
+You can upgrade from any version 5.x release to a later 5.x release. 
 
 ## Upgrading from version 4 to version 5
 
-Upgrades from PGD 4 to PGD 5 are supported from version 4.3.0. For older versions, upgrade to 4.3.0 before upgrading to 5. See [Upgrading within 4](/pgd/4/upgrades/upgrade_paths/#upgrading-within-version-4) for more information. After upgrading to 4.3.0 or later, the following combinations are allowed.
-
-| 4.3.0 | 4.3.0-1 | 4.3.1 | 4.3.1-1 | Target PGD version |
-|-------|---------|-------|---------|--------------------|
-| ✓     |  ✓      |   ✓   |    ✓    | 5.2.0              |
-| ✓     |  ✓      |   ✓   |         | 5.1.0              |
-| ✓     |  ✓      |       |         | 5.0.1              |
-| ✓     |         |       |         | 5.0.0              |
-
-
+Generally, we recommend you upgrade to the latest version 4 release, before
+upgrading to the latest version 5 release. Upgrades from PGD 4 to PGD 5 are
+ supported from version 4.3.0 or later. See [Upgrading within
+4](/pgd/4/upgrades/upgrade_paths/#upgrading-within-version-4) for more
+information. After upgrading t the following upgrade paths are
+possible.
+
+| From version | To version |
+| ---- | -- |
+| 4.3.0 | 5.0.0 or later |
+| 4.3.0-1 | 5.0.1 or later |
+| 4.3.1 | 5.1.0 or later |
+| 4.3.1-1 | 5.2.0 or later |
```

Comment:
Pretty much the same here, I'd just scratch the `4.3.1-1` line.

Reply:
So just pull the +p releases lines leaving only the BDR version? I asm as 4.3.0+p1 goes the 5.0.1 not 5.0.0

Reply:
Like I said in the other change, the 4.3.1 -> 4.3.1+p1 bumps the target version from 5.1.0 to 5.2.0

Is that (a) correct and (b) if it is, shouldn't we keep it in?